generator/lkml_update.py

```diff
@@ -0,0 +1,49 @@
+"""An updated lkml parser to handle explore queries."""
+from typing import List, Union
+
+from lkml.keys import KEYS_WITH_NAME_FIELDS
+from lkml.simple import DictParser
+from lkml.tree import BlockNode, DocumentNode, ListNode, PairNode
+
+
+def dump(obj: dict) -> str:
```

Comment:
Can you give this a slightly more descriptive name, like `lkml_dump`? It's hard to see where this is coming from in the consumers.

Reply:
What if instead we import the file? Then use `lkml_update.dump`.